### PR TITLE
Add TruffleRuby support

### PR DESCRIPTION
**Pull request by sponomarev**
Follow up for #26

### CODE REVIEW

File: lib/lite_cable/server/client_socket/base.rb [hunk 1]
```diff
@@ -142,10 +142,10 @@ def each_frame
           framebuffer = WebSocket::Frame::Incoming::Server.new(version: version)
 
           while IO.select([socket])
-            if socket.respond_to?(:recvfrom)
-              data, _addrinfo = socket.recvfrom(2000)
+            if socket.respond_to?(:recv)
+              data = socket.recv(2000)
```
Comment (sponomarev) [inline lib/lite_cable/server/client_socket/base.rb]: We don't need resolved address for TCP socket and https://github.com/oracle/truffleruby/issues/1550

**Comment by sponomarev** [comment]
@palkan Rebased over the latest master

**Comment by sponomarev** [comment]
Rebase over the latest master

**Comment by palkan** [comment]
🎉

**Pull request merged**